charts/chronograf/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: chronograf
-version: 1.1.19
+version: 1.1.20
 appVersion: 1.8.8
```

Comment:
1.8.9 is now out too.  Let's bump that forward as well.

Reply:
bumped in 9310ec2